commands.go

```diff
@@ -1113,7 +1119,11 @@ func (c *commandable) zRangeByScore(key string, opt ZRangeByScore, withScores bo
 }
 
 func (c *commandable) ZRangeByScore(key string, opt ZRangeByScore) *StringSliceCmd {
-	return c.zRangeByScore(key, opt, false)
+	return c.zRangeByScore(key, opt, false, false)
+}
+
+func (c *commandable) ZRangeByLex(key string, opt ZRangeByScore) *StringSliceCmd {
```

Comment:
I would also prefer to rename `ZRangeByScore` to `ZRangeByOpts`, but it would break the API. @vmihailenco: thoughts?